lib/signs/utilities/audio.ex

```diff
@@ -179,21 +158,13 @@ defmodule Signs.Utilities.Audio do
 
   defp log_crowding(new_audios, sign_id) do
     Enum.each(new_audios, fn
-      %{
+      %Audio.Approaching{
         trip_id: trip_id,
         crowding_description: crowding_description,
-        route_id: "Orange",
-        __struct__: audio_type
-      }
-      when audio_type in [Audio.Approaching, Audio.TrainIsArriving] ->
-        announcement_type =
-          case audio_type do
-            Audio.Approaching -> "approaching"
-            Audio.TrainIsArriving -> "arrival"
-          end
-
+        route_id: "Orange"
+      } ->
         Logger.info(
-          "crowding_log: announcement_type=#{announcement_type} trip_id=#{trip_id} sign_id=#{sign_id} crowding_description=#{inspect(crowding_description)}"
+          "crowding_log: announcement_type=approaching trip_id=#{trip_id} sign_id=#{sign_id} crowding_description=#{inspect(crowding_description)}"
```

Comment:
Is `announcement_type` used in some existing report/dashboard or can we get rid of it?

Reply:
I didn't look into it, so this seemed like the safest change. Definitely worth looking at, at some point.